platform/linux/mbedtls/network_mbedtls_wrapper.c

```diff
@@ -286,73 +286,116 @@ IoT_Error_t iot_tls_connect(Network *pNetwork, TLSConnectParams *params) {
 	return (IoT_Error_t) ret;
 }
 
+/* When this much time has elapsed after receiving MBEDTLS_ERR_SSL_WANT_READ
+ * or MBEDTLS_ERR_SSL_WANT_WRITE, then iot_tls_write will return
+ * NETWORK_SSL_WRITE_TIMEOUT_ERROR. */
+#define IOT_SSL_WRITE_RETRY_TIMEOUT_MS 10
```

Comment:
Why is this decided to be provided as a MACRO in .c file? Can't this be set in a config header?

Reply:
I decided to do this because `IOT_SSL_READ_TIMEOUT` was also originally defined in the .c file. I think the reasoning here was to provide reasonable defaults to the application. That being said, I've wrapped these in `#ifndef` so that they can be configured in `aws_iot_config.h`.